galaxyui/src/app/content-detail/repository/repository.component.html

```diff
@@ -1,53 +1,65 @@
 <div class="row repository-container">
-    <div class="col-sm-4">
-        <div class="title-box">
-            <div class="icon-container">
-                <span class="fa fa-5x" [ngClass]="repositoryView.iconClass" tooltip="{{ repositoryView.tooltip }}"></span>
-            </div>
-            <div class="title-container">
-                <div class="title">{{ repositoryView.name }}</div>
-                <div class="description">{{ repositoryView.description }}</div>
-                <a class="namespace"
-                   [href]=""
-                   (click)="viewAuthor()"
-                   tooltip="View the Author's page">
-                   <img [src]="repositoryView.avatarUrl" > {{ repositoryView.displayName }}</a>
-            </div>
+
+    <div class="col-lg-2">
+        <div class="namespace-img">
+            <img class="img-wrapper" tooltip="View Author's Profile" [src]="namespace.avatar_url" (click)="viewAuthor()">
+            <a (click)="viewAuthor()"><span class="fa fa-user"></span> {{ namespace.name }}</a>
```

Comment:
We need to fix these links as well, but I think that would be easier to do in a separate PR.